src/test/java/com/hocs/test/glue/DraftResponseStepDefs.java

```diff
@@ -36,25 +39,69 @@
 
     Qa qa;
 
-    @When("^I complete the initial draft stage$")
-    public void completeInitialDraftStage() {
-        dataInput.selectTeam1();
-        successfulCaseCreation.clickSessionVariableViaLinkText();
-        draftingTeamDecision.clickAcceptInitialDraftDecision();
-        page.clickContinueButton();
-        draftingTeamDecision.clickDraftingResponseLetter();
-        page.clickContinueButton();
-        draft.clickAddDocumentsButton();
-        draft.selectDocumentTypeByIndex(1);
-        addDocuments.uploadDocument();
-        page.clickAddButton();
-        page.clickContinueButton();
-        // offline QA decision NO
-        qa.clickOfflineQANoRadioButton();
-        page.clickContinueButton();
-        //qaResponse.clickOfflineQANoButton();
-        System.out.println("I have completed the initial draft and am QA'ing the case");
-        // Should belong to QAResponse StepDefs vv
+    Teamqueue teamqueue;
+
+    @When("^Initial draft stage \"([^\"]*)\"$")
+    public void completeInitialDraftStage(String caseType) {
+        switch (caseType.toUpperCase()){
+            case "DCU MIN":
+                homepage.findMyInitialDraftCase();
+                successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
+                homepage.selectAllocationUserByVisibleText("Danny Large (danny.large@ten10.com)");
+                homepage.selectMyCases();
```

Comment:
maybe you could split this logic out into separate methods which do most of the logic. ie one method to do allt he homepage stuff, then another method to do the next main chunk ect. just a thought